src/System Application/Test Library/File System/app.json

```diff
@@ -0,0 +1,39 @@
+{
+    "id":  "f188754b-3ffb-443a-9507-f5fbdae3af2c",
+    "name":  "File System Test Library",
+    "publisher":  "Microsoft",
+    "brief":  "Test library for the File System module",
+    "description":  "Test library for the File System module",
+    "version":  "25.0.0.0",
```

Comment:
I think you may have to update those versions too 😉

Reply:
Sorry not my day. Should be correct now i hope.